ci-operator/step-registry/assisted/nutanix/setup/assisted-nutanix-setup-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: assisted-nutanix-setup
+  steps:
+    # Configure nutanix parameters
+    - ref: ipi-conf-nutanix-context
+    # Creating the nutanix machine image for the assisted-test-infra machine
+    # - ref: assisted-nutanix-setup-template
```

Comment:
I'd like to keep it so that the step would be used again when Nutanix Packer plugin would know how to properly replace images (or fixed to do that manually)

Reply:
sounds good
can you add a TODO comment explaining that?